src/app/layout.js

```diff
@@ -13,11 +11,7 @@ export const metadata = {
 export default function RootLayout({ children }) {
   return (
     <html lang='es'>
-      <body className={montserrat.className}>
-        <NavBar />
-        {children}
-        <Footer />
-      </body>
```

Comment:
Having the `<NavBar />` and the `<Footer />` here is key to make these two component render through the entire web application.